lib/runtime/application.ts

```diff
@@ -195,12 +198,24 @@ export default class Application extends Addon {
    * config/environment.js
    */
   private async createServer(port: number): Promise<void> {
-    await new Promise((resolve) => {
+      await new Promise((resolve) => {
```

Comment:
this should be de-tabbed 😛 

Also, I don't know how this would affect things, but because we're pushing a new Promise to the drainers array down below, would there be conflict between the two `resolve` callbacks?